migrate.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
```

Comment:
A bash script as important as this one should have the 3 bash "strict-mode" settings added to it right after the sharpbang:

```bash
set -e
set -u
set -o pipefail
```